tools/DevDiagnostics/DevHome.DevDiagnostics/Strings/en-us/Resources.resw

```diff
@@ -1278,4 +1278,24 @@ Dump File creation time: {1}</value>
     <value>(No insights found)</value>
     <comment>Placeholder text on the Insights page when the insights list is empty.</comment>
   </data>
+  <data name="InsightProcessExitMissingDependenciesTitle" xml:space="preserve">
+    <value>Process exited due to missing files</value>
+    <comment>Title of an insight informing the user of a reason why a process exited</comment>
+  </data>
+  <data name="InsightProcessExitMissingDependencies" xml:space="preserve">
+    <value>Process {0} (pid: {1,6}) exited with error code {2}. Enabling loader snaps and trying to launch the process again can help diagnose why the app exited.</value>
```

Comment:
I'm reading this as "something is now happening to enable loader snaps". Is that accurate - or is this a prompt to the user to hit a button in order to enable loader snaps?

Reply:
See the screen shot above. It's a prompt to the user to hit the button.

Reply:
Quite: but that's not how it reads... Instead of "Enabling..." and "trying...", perhaps "Enable..." and "try..."?

Reply:
Got it.